docs/products/m3db/reference/terminology.rst

```diff
@@ -0,0 +1,19 @@
+Terminology for M3
+==================
+
+Time series
+    unique series of data identified by a set of labels, containing value(s) for specific points in time
+
+Point
+    (time, value) entry in single time series 
+
+Namespace
+    retention-period limited view of the database, with (most of the time) all time series included; either unaggregated (all points), or aggregated at some resolution (at most only one point per resolution duration)
```

Comment:
I would rephrase this:

"analogous to tables in other databases, contain a view of time series dataset limited to the retention-period". It can be **unaggregated** (all points), or **aggregated** at some resolution (at most only one point per resolution duration)

Reply:
adapted slightly but thanks!